plugin/v2/plugin.go

```diff
@@ -95,6 +96,7 @@ func (p *Plugin) InitPlugin() error {
 	err = json.NewDecoder(dt).Decode(&p.PluginObj.Manifest)
 	dt.Close()
 	if err != nil {
+		logrus.Errorf("InitPlugin: Parsing manifest failed due to %v", err)
```

Comment:
@anusha-ragunathan won't this be logged twice ? once here and once in the calling function ?